client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -213,11 +232,62 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 					fromSite={ fromSite }
 					urlData={ fromSiteData ?? undefined }
 					stepNavigator={ stepNavigator }
+					renderHeading={ ! useContainerV2 }
 				/>
 			);
 		};
 
 		const importJob = getImportJob( importer );
+
+		if ( useContainerV2 ) {
+			const importerData = getImportDragConfig( importer, stepNavigator?.supportLinkModal );
+			const showHeading =
+				importJob?.importerState === appStates.MAP_AUTHORS ||
+				importJob?.importerState === appStates.READY_FOR_UPLOAD ||
+				importJob?.importerState === appStates.UPLOAD_PROCESSING ||
+				importJob?.importerState === appStates.UPLOADING;
+
+			const showBackButton = importJob?.importerState !== appStates.IMPORT_SUCCESS;
+			return (
+				<>
+					<QuerySites siteId={ siteId } />
+					<DocumentHead title={ __( 'Import your site content' ) } />
+					<Interval onTick={ fetchImporters } period={ EVERY_FIVE_SECONDS } />
+					<Step.CenteredColumnLayout
+						className={ clsx(
+							'import__onboarding-page',
+							'importer-wrapper',
+							'step-container-v2--site-migration-identify',
+							'import__onboarding-page--redesign',
+							{
+								[ `importer-wrapper__${ importer }` ]: !! importer,
+							}
+						) }
+						columnWidth={ 6 }
+						topBar={
+							<Step.TopBar
+								leftElement={ showBackButton ? <Step.BackButton onClick={ onGoBack } /> : null }
+								rightElement={
+									! showBackButton ? (
```

Comment:
I think it would be better to create a second constant to indicate whether we should skip the button or not.
I am assuming the relation here is just a coincidence, please let me know I am wrong.

Reply:
We don't show the back button when the import succeeds, so the only place to go is forward. Technically it's the same but I agree that using a separate const is clearer